### PR TITLE
Fix blue showing as red

### PR DESCRIPTION
**Pull request by peregrineshahin**
make it consistent with the db state variable.

### CODE REVIEW

**Comment by vdbergh** [comment]
I don't think we should say in the event log that the result of a test is blue. It is a passed test so the result is green. In the gui it is colored blue to draw attention to the fact that it is a simplification. But this is not necessary for the event log. 

Perhaps it would be better to just say passed/failed?

**Comment by peregrineshahin** [comment]
I'm not sure about calling it a green, since in that case we would need to flag blue ones as `isGreen`, pretty sure passed/failed are good enough but since we can argue that yellow is also a failed test but we mention it and flag it in the db then I'm not sure why not blue..

**Comment by vdbergh** [comment]
> I'm not sure about calling it a green, since in that case we would need to flag blue ones as `isGreen`, pretty sure passed/failed are good enough but since we can argue that yellow is also a failed test but we mention it and flag it in the db then I'm not sure why not blue..

Yes I think mentioning yellow in the event log was also a big mistake. I only did it because one can currently search for yellow tests in the gui, but in fact yellow has little statistical meaning.

Therefore I am proposing the labels `passed` and `failed` in the event log, instead of the color labels.

**Comment by peregrineshahin** [comment]
I'm a bit in favor of `accepted` and `rejected` since `failed` can mislead the reader because we can have failed tests for the wrong benches or if the code has errors, probably more consistent with the state variable also, do you think that will do?

probably with one-liner:
```result = state``` ?

**Comment by peregrineshahin** [comment]
Let me know please if fixed games please can either be green or red because I'm not sure, so that we don't calculate based on los and have two sources of truth (if the bounds changed requires changing here) or if we have any simpler way to do it.

**Comment by vdbergh** [comment]
> Let me know please if fixed games can either be green or red because I'm not sure, so that we don't calculate based on los and have two sources of truth (if the bounds changed requires changing here) or if we have any simpler way to do it.

I would say that fixed is accepted if it currently labeled green and rejected otherwise. For SPRT we can indeed just take the state!

**Comment by vdbergh** [comment]
I am happy with the new logic!

**Comment by vdbergh** [comment]
Actually I see that also the title of the notification has to be changed. From `XXX by XXX finished <color>` to `XXX by XXX was <state>`.

**Comment by ppigazzini** [comment]
DEV updated, running some tests with expected outcome.

**Comment by ppigazzini** [comment]
Duplicated notification for the first test, Events log is fine, though.

image

image

**Comment by vdbergh** [comment]
The duplicate notifications are the result of a benign race condition. This can be partially fixed with weblocks but there is still the issue that the communication between tabs using local storage is not instantaneous. A more serious issue is that I think that when IOS goes to sleep, writes to local storage may simply be discarded and so they are never seen by another tab. This seems unfixable.

In short I have not succeeding in eliminating the problem on IOS. I will have to settle for mitigation.

**Comment by ppigazzini** [comment]
If the text of the notifications meets the expectation , please squash the commits.

**Comment by ppigazzini** [comment]
PROD updated, thank you @peregrineshahin :)
